Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -876,14 +1019,388 @@ def fetch_incidents(client, last_run, first_fetch_time, min_severity):
+        'revoked': args.get('revoked'),
+        'confidence': arg_to_number(args.get('confidence')),
+        'threatTypes': argToList(args.get('threat_types')),
+        'includeDisabled': args.get('includeDisabled', ''),
+        'source': args.get("source", "Azure Sentinel"),
+        'threatIntelligenceTags': argToList(args.get('tags')),
+        'validFrom': args.get('valid_from', ''),
+        'validUntil': args.get('valid_until', ''),
+        'createdByRef': args.get('created_by', ''),
+    }
+
+    indicator_type = args.get('indicator_type')
+    if indicator_type == 'ipv4':
+        indicator_type = 'ipv4-address'
+    elif indicator_type == 'ipv6':
+        indicator_type = 'ipv6-address'
+    elif indicator_type == 'domain':
+        indicator_type = 'domain-name'
+
+    data['patternTypes'] = indicator_type
+
+    if indicator_type == 'file':
+        hash_type = args.get('hash_type')
+        data['hashType'] = hash_type
+        data['pattern'] = f"[file:hashes.'{hash_type}' = {value}]"
+    else:
+        data['pattern'] = f'[{indicator_type}:value = {value}]'
+
+    data['killChainPhases'] = []
+
+    kill_chains = argToList(args.get('kill_chains', []))
+    if kill_chains:
+        for kill_chain_phase in kill_chains:
+            data['killChainPhases'].append(
+                {'killChainName': kill_chain_phase,
+                 'phaseName': kill_chain_phase})
+
+    remove_nulls_from_dictionary(data)
+
+    return data
+
+
+def update_data_of_indicator(new_ind_data, original_ind_data):
+    original_extracted_data = extract_original_data_from_indicator(original_ind_data.get("properties"))
+    new_data = get_data_for_new_ind(new_ind_data)
+
+    original_extracted_data.update(new_data)
+
+    return original_extracted_data
+
+
+def extract_original_data_from_indicator(original_data):
+    extracted_data = {
+
+        'description': original_data.get('description', ''),
+        'revoked': original_data.get('revoked', ''),
+        'confidence': arg_to_number(original_data.get('confidence')),
+        'threatTypes': argToList(original_data.get('threatTypes')),
+        'killChainPhases': argToList(original_data.get('killChainPhases')),
+        "threatIntelligenceTags": argToList(original_data.get("threatIntelligenceTags")),
+        "validFrom": original_data.get("validFrom", ''),
+        "validUntil": original_data.get("validUntil", ''),
+        "createdByRef": original_data.get("createdByRef", ''),
+        "created": original_data.get('created', ''),
+        "externalId": original_data.get("externalId"),
+        'displayName': original_data.get("displayName"),
+        'source': original_data.get("source")
+    }
+
+    remove_nulls_from_dictionary(extracted_data)
+    return extracted_data
+
+
+def list_threat_indicator_command(client, args):
+    """
+    returns a list of threat indicators
+
+    Returns: a list of threat indicators
+    """
+    url_suffix = 'threatIntelligence/main/indicators'
```

Comment:
```suggestion
def list_threat_indicator_command(client, args):
    url_suffix = 'threatIntelligence/main/indicators'
```
when a function is straightforward, better to avoid documenting it.
and a general note, instead of the "returns a list of threat indicators" in the first line, we provide all arguments information including type and a short description.